src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -138,27 +126,184 @@ public static void setDebug(boolean debug) {
     /**
      * The default duration in milliseconds spent not scanning between each bluetooth scan cycle when no ranging/monitoring clients are in the foreground
      */
-    public static final long DEFAULT_BACKGROUND_BETWEEN_SCAN_PERIOD = 5*60*1000;
+    public static final long DEFAULT_BACKGROUND_BETWEEN_SCAN_PERIOD = 5 * 60 * 1000;
 
     private long foregroundScanPeriod = DEFAULT_FOREGROUND_SCAN_PERIOD;
     private long foregroundBetweenScanPeriod = DEFAULT_FOREGROUND_BETWEEN_SCAN_PERIOD;
     private long backgroundScanPeriod = DEFAULT_BACKGROUND_SCAN_PERIOD;
     private long backgroundBetweenScanPeriod = DEFAULT_BACKGROUND_BETWEEN_SCAN_PERIOD;
 
+    private static Logger sLogger = Loggers.empty();
```

Comment:
I would still like the default logger to be Loggers.androidLogger().  Most Android developers will be looking at LogCat during their development process, and it is important for them to see errors and warnings produced by the library.  If the default is Loggers.empty() then people won't know when things go wrong unless they have taken action to set up an alternate logger.